Plugins/Public/mobiledocking_plugin/Main.cpp

```diff
@@ -111,37 +111,6 @@ uint GetInstalledModules(uint iClientID)
 	return modules;
 }
 
-void LogCheater(uint client, const wstring &reason)
-{
-	CAccount *acc = Players.FindAccountFromClientID(client);
-
-	if (!HkIsValidClientID(client) || !acc)
-	{
-		AddLog("ERROR: invalid parameter in log cheater, clientid=%u acc=%08x reason=%s", client, acc, wstos(reason).c_str());
-		return;
-	}
-
-	// Set the kick timer to kick this player. We do this to break potential
-	// stack corruption.
-	HkDelayedKick(client, 1);
-
-	// Ban the account.
-	flstr *flStr = CreateWString(acc->wszAccID);
-	Players.BanAccount(*flStr, true);
-	FreeWString(flStr);
-
-	// Overwrite the ban file so that it contains the ban reason
-	wstring wscDir;
-	HkGetAccountDirName(acc, wscDir);
-	string scBanPath = scAcctPath + wstos(wscDir) + "\\banned";
-	FILE *file = fopen(scBanPath.c_str(), "wbe");
-	if (file)
-	{
-		fprintf(file, "Autobanned by Docking Module\n");

```

Comment:
Did a search, no matches. Tested locally and there does not seem to be a way to trigger this behaviour in an abusive way.